src/components/SummaryBox/SummaryBoxHeading/SummaryBoxHeading.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface SummaryBoxHeadingProps {
+  heading: string
+  className?: string
+  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
+}
+
+export const SummaryBoxHeading = ({
+  heading,
+  className,
+  headingLevel,
```

Comment:
You could give `headingLevel` a default here rather than handle that below with `headingLevel = h3`.  That might be more explicit for someone reading this code.